<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>monitorClass</class>
 <widget class="QMainWindow" name="monitorClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1332</width>
    <height>773</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>monitor</string>
  </property>
  <property name="windowIcon">
   <iconset resource="monitor.qrc">
    <normaloff>:/monitor/Resources/timg.jpg</normaloff>:/monitor/Resources/timg.jpg</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="picture">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>54</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QPushButton" name="open_btn">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>560</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>打开</string>
    </property>
   </widget>
   <widget class="QLabel" name="picture2">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>20</y>
      <width>54</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QPushButton" name="close_btn">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>640</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>关闭</string>
    </property>
   </widget>
   <widget class="QPushButton" name="camera2Open_btn">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>570</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>摄像头2打开</string>
    </property>
   </widget>
   <widget class="QPushButton" name="cameraOpen_btn">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>570</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>摄像头全打开</string>
    </property>
   </widget>
   <widget class="QPushButton" name="cameraClose_btn">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>570</y>
      <width>75</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>摄像头关闭</string>
    </property>
   </widget>
   <widget class="QPushButton" name="camera1Open_btn">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>570</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>摄像头1打开</string>
    </property>
   </widget>
   <widget class="QPushButton" name="link_btn">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>600</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>重连</string>
    </property>
   </widget>
   <widget class="QSlider" name="moveDir_vSlider">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>530</y>
      <width>31</width>
      <height>81</height>
     </rect>
    </property>
    <property name="maximum">
     <number>2</number>
    </property>
    <property name="pageStep">
     <number>1</number>
    </property>
    <property name="value">
     <number>1</number>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QSlider" name="horizontalSlider">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>560</y>
      <width>121</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <number>180</number>
    </property>
    <property name="value">
     <number>90</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="speed0_btn">
    <property name="geometry">
     <rect>
      <x>960</x>
      <y>550</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QPushButton" name="speed1_btn">
    <property name="geometry">
     <rect>
      <x>1010</x>
      <y>550</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="speed2_btn">
    <property name="geometry">
     <rect>
      <x>1060</x>
      <y>550</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="speed3_btn">
    <property name="geometry">
     <rect>
      <x>1110</x>
      <y>550</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>3</string>
    </property>
   </widget>
   <widget class="QPushButton" name="speed4_btn">
    <property name="geometry">
     <rect>
      <x>1160</x>
      <y>550</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>4</string>
    </property>
   </widget>
   <widget class="QPushButton" name="speed5_btn">
    <property name="geometry">
     <rect>
      <x>1210</x>
      <y>550</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>5</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="left_lEdit">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>640</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="picture2_2">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>640</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>方向1：</string>
    </property>
   </widget>
   <widget class="QLabel" name="picture2_3">
    <property name="geometry">
     <rect>
      <x>920</x>
      <y>640</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>方向1：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="right_lEdit">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>960</x>
      <y>640</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="picture2_4">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>640</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>PWM1：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="leftDuty_lEdit">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>640</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="picture2_5">
    <property name="geometry">
     <rect>
      <x>1020</x>
      <y>640</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>PWM2：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="rightDuty_lEdit">
    <property name="geometry">
     <rect>
      <x>1060</x>
      <y>640</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="rightDuty_lEdit_2">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>1060</x>
      <y>670</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="left_lEdit_2">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>670</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="picture2_6">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>670</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>方向1：</string>
    </property>
   </widget>
   <widget class="QLabel" name="picture2_7">
    <property name="geometry">
     <rect>
      <x>920</x>
      <y>670</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>方向1：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="right_lEdit_2">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>960</x>
      <y>670</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="picture2_8">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>670</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>PWM1：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="leftDuty_lEdit_2">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>670</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="picture2_9">
    <property name="geometry">
     <rect>
      <x>1020</x>
      <y>670</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>PWM2：</string>
    </property>
   </widget>
   <widget class="QPushButton" name="check_btn">
    <property name="geometry">
     <rect>
      <x>1130</x>
      <y>670</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>查询</string>
    </property>
   </widget>
   <widget class="QPushButton" name="setPWM_btn">
    <property name="geometry">
     <rect>
      <x>1130</x>
      <y>640</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>设置</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="caremaPort1_lEdit">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>630</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="picture2_10">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>630</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>摄像头1端口</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="left_lEdit_4">
    <property name="geometry">
     <rect>
      <x>-530</x>
      <y>620</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="caremaPort2_lEdit">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>630</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="picture2_11">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>630</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>摄像头2端口</string>
    </property>
   </widget>
   <widget class="QPushButton" name="port1_btn">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>660</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>设置</string>
    </property>
   </widget>
   <widget class="QPushButton" name="port2_btn">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>660</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>设置</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1332</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="monitor.qrc"/>
 </resources>
 <connections/>
</ui>
